synthtool/gcp/templates/java_library/.kokoro/presubmit/samples.cfg

```diff
@@ -11,14 +11,20 @@ env_vars: {
   value: "samples"
 }
 
+# TODO: remove this after we've migrated all tests and scripts
 env_vars: {
-    key: "GCLOUD_PROJECT"
-    value: "gcloud-devel"
+  key: "GCLOUD_PROJECT"
+  value: "java-docs-samples-testing"
 }
 
 env_vars: {
-    key: "GOOGLE_APPLICATION_CREDENTIALS"
-    value: "keystore/73713_java_it_service_account"
+  key: "GOOGLE_CLOUD_PROJECT"
+  value: "java-docs-samples-testing"
+}
+
+env_vars: {
+  key: "GOOGLE_APPLICATION_CREDENTIALS"
+  value: "keystore/73713_java_it_service_account"
```

Comment:
I have no idea how keystore works. Does this service account include the permission to `java-docs-samples-testing` project? or Does this have to be updated?

Reply:
It does for some APIs which we will need to grant permissions for.